### PR TITLE
Jon/how to update docs

### PR DESCRIPTION
**Pull request by jonmatthis**
made a sloppy first draft of a how-to-update docs how to guide

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+.idea
```
Comment (jonmatthis) [inline .gitignore:131]: why did you do this?

Comment (jonmatthis) [reply .gitignore:131]: oh i see

File: docs/index.md [hunk 2]
```diff
@@ -1,19 +1,9 @@
-# Welcome to MkDocs
+# Welcome to the Knowledge Base for the HuMon Research Lab 
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+The page is built via the `material` theme of mkdocs
 
-test hello hi :D 
+mkdocs lives here -
```
Comment (jonmatthis) [inline docs/index.md]:
```suggestion
Documentation for `mkdocs` lives here - 
```

File: README.md [hunk 3]
```diff
@@ -4,9 +4,9 @@
 
 [![mkdocs_material_deploy](https://github.com/HuMoN-Research-Lab/humon.github.io/actions/workflows/mkdocs_material_deploy.yml/badge.svg)](https://github.com/HuMoN-Research-Lab/humon.github.io/actions/workflows/mkdocs_material_deploy.yml)
 
-
 based on `MkDocs Material` theme
 
 Check here -  https://squidfunk.github.io/mkdocs-material/  - for instructions on how to set it up, change the appearance, etc etc 
 
 build knowledge base/docs based on https://diataxis.fr
+
```
Comment (trentwirth) [inline README.md:12]: this is line 12

File: docs/index.md [hunk 4]
```diff
@@ -1,19 +1,9 @@
-# Welcome to MkDocs
+# Welcome to the Knowledge Base for the HuMon Research Lab 
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+The page is built via the `material` theme of mkdocs
 
-test hello hi :D 
+Documentation for `mkdocs` lives here -
```
Comment (trentwirth) [inline docs/index.md:5]: this is a review, I'm FURIOUS

Comment (trentwirth) [reply docs/index.md:5]: (I'm not really)

**Review by trentwirth — CHANGES_REQUESTED** [review]
👎

File: docs/index.md (hunk 4 above)
Comment (jonmatthis) [reply docs/index.md:5]: omg i'm so sorry. I fixed it 😬

**Review by trentwirth — APPROVED** [review]
✨ 💀 ✨

**Pull request merged**